reference/commands/inspect.rst

```diff
@@ -23,3 +23,127 @@ conan inspect
                             -vvv or -vtrace
       --logger              Show the output with log format, with time, type and
                             message.
+
+Inspect default and json output
```

Comment:
Please tag ``conan inspect`` as preview, it was always a bit problematic command